packages/flutter_tools/lib/src/cache.dart

```diff
@@ -207,7 +207,7 @@ class FlutterEngine {
   // Return a list of (cache directory path, download URL path) tuples.
   List<List<String>> _getToolsDirs() {
     if (Platform.isMacOS)
-      return <List<String>>[<String>['darwin-x64', 'darwin-x64/artifacts.zip']];
+      return <List<String>>[<String>['mac_release', 'mac_release/artifacts.zip']];
```

Comment:
It would be great to use the `-` here for consistency w/ the other buckets. Ideally we'd be able to synthesize the bucket name just knowing the host or target platform and the build mode.

```

android-arm-profile/
android-arm-release/
android-arm/
android-x64/
android-x86/

ios_debug/
ios_profile/
ios_release/

linux-x64/
mac_release/
```